rb_ws/src/buggy/scripts/auton/autonsystem.py

```diff
@@ -61,7 +61,9 @@ def __init__(self,
         self.local_controller = local_controller
         self.brake_controller = brake_controller
 
-        self.path_planner = PathPlanner(global_trajectory)
+        # left_curb = Trajectory(json_filepath="/rb_ws/src/buggy/paths/outside_curb_smooth.json")
+        left_curb = None
```

Comment:
This should be a part of the launch file to make sure that we can add a curb from the launch file